BUILD

```diff
@@ -67,6 +67,8 @@ java_binary(
     main_class = "grakn.console.GraknConsole",
     runtime_deps = [":console"],
     visibility = ["//:__pkg__"],
+    resource_strip_prefix = "conf/logback",
+    resources = ["//conf/logback:logback.xml"]
```

Comment:
why are we providing logback now? console should only print to stdout, right?

Reply:
this is because running this bazel target with `bazel run` leads to alllll the netty logs being printed. horrible for debugging!

Reply:
makes sense.